pkg/controller/operators/catalog/operator.go

```diff
@@ -75,6 +76,8 @@ type Operator struct {
 	resolver               resolver.Resolver
 	reconciler             reconciler.RegistryReconcilerFactory
 	csvProvidedAPIsIndexer map[string]cache.Indexer
+	strategy               *scoped.ClientStrategy
```

Comment:
non-blocking: how about "attenuator" or "clientAttenuator"? Attenuation is a relatively standard term for reducing the level of permission.

Reply:
yes, that makes sense, it is now `scoped.ClientAttenuator`